src/vs/editor/contrib/inlineCompletions/browser/inlineCompletionsController.ts

```diff
@@ -199,6 +203,24 @@ export class InlineCompletionsController extends Disposable {
 		this._register(new InlineCompletionsHintsWidget(this.editor, this.model, this.instantiationService));
 	}
 
+	private provideScreenReaderHint(): void {
+		const showHoverKeybinding = this._keybindingService.lookupKeybinding('editor.action.showHover');
+		const accessibleViewKeybinding = this._keybindingService.lookupKeybinding('editor.action.accessibleView');
+		if (this.configurationService.getValue('accessibility.verbosity.inlineCompletions')) {
+			let hint: string | undefined;
+			if (showHoverKeybinding && accessibleViewKeybinding) {
+				hint = localize('showBothHints', "View more actions ({0}) or inspect this in the accessible view ({1})", showHoverKeybinding.getAriaLabel(), accessibleViewKeybinding.getAriaLabel());
+			} else if (showHoverKeybinding) {
+				hint = localize('showHoverHint', "View more actions ({0})", showHoverKeybinding.getAriaLabel());
+			} else if (accessibleViewKeybinding) {
+				hint = localize('showAccessibleViewHint', "Inspect this in the accessible view ({0})", accessibleViewKeybinding.getAriaLabel());
+			}
+			if (hint) {
+				status(hint);
```

Comment:
Does it work to call status and alert at the same time? What does status do?

Reply:
`status` is queued so it happens after the `alert` which is critical and what we want here